default/stringtables/en.txt

```diff
@@ -4628,6 +4628,14 @@ Species Traits
 SPECIES_TRAITS_TEXT
 Non-exhaustive list of Species Traits:
 
+SYSTEM_SUPERIORITY_TITLE
+System Superiority
+
+SYSTEM_SUPERIORITY_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system unoccupied by hostile fleets, it establishes system superiority. Once it has been established, supply is interrupted for hostile empires, industry of hostile planets can only be used on the planet it is generated on, and ground troop regeneration is stopped. Also, all starlane entries in the occupied system become guarded by the blockading force, so if hostile forces arrive, they can only exit by the starlane entry by which they arrived.
+If two hostile fleets arrive at an unoccupied system at the same time, no system superiority is established until only one side remains and the other one has either been destroyed or retreated. Until then, both empires will maintain supply for their empire.
```

Comment:
I think "blockade" is a well known concept with a much more narrow and applicable meaning than the rather vague "system superiority".  I could much more readily consider the term "system control" rather than "system superiority", but still the term "blockade" seems much better to me.  Can you describe any particular problem you see with using the term "blockade"?

Also, I think the clause 
```industry of hostile planets can only be used on the planet it is generated on``` 
would be a little better to refer to the hostile planets specifically in the blockaded system, and maybe with a little more adjustment, like 
```industry generated on hostile planets within the blockaded system can only be used on the respective planet.```